src/icons/explicitHydrogen.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill="none" d="M0 0h24v24H0z" data-name="Trim Area"/><path d="M11 11H5V5H3v14h2v-6h6v6h2V5h-2v6zm8-4V5h-2v2h-2v2h2v2h2V9h2V7h-2zm-4 8h6v2h-6z" fill="#505050"/></svg>
```

Comment:
This doesn't work for some reason... Anyway I'll ask Robert to give me an SVG icon file.

Reply:
strange... it works after a server restart. Anyway will wait for Robert.